src/igraph/__init__.py

```diff
@@ -1919,9 +1919,6 @@ def from_networkx(cls, g):
 
         @param g: networkx Graph or DiGraph
         """
-        import networkx as nx
```

Comment:
Did you mean to remove the `networkx` import as well, or was it an accidental casualty?

Reply:
Or is it the case that we don't access anything form the networkx namespace directly (we only call methods of the `g` argument), this is not needed? Just asking for my Python education :-)

Reply:
@iosonofabio is right, you don't need the import since you don't use anything from `nx` explicitly in this function. I just wanted to say that a Python linter usually detects issues like this, but then I realized that most linters are unusable on the source code of `python-igraph` yet because there are so many warnings that should be cleaned up. (Most of them in `__init__.py` are warnings about stuff being imported but unused, and in 99% of the cases it should be solved simply by including these explicity in `__all__` to declare that they are part of the public API).